src/SegmentedString.chpl

```diff
@@ -1524,7 +1536,35 @@ module SegmentedString {
     The SegFunction called by computeOnSegments for isdigit
   */
   inline proc stringIsDigit(ref values, rng) throws {
-    return interpretAsString(values, rng, borrow=true).isDigit();
+    use In1d;
+    const special_digits = "⁰¹²³⁴⁵⁶⁷⁸⁹₀₁₂₃₄₅₆₇₈₉";
+    const myString = interpretAsString(values, rng, borrow=true);
```

Comment:
we're mixing `_` and camelCase. We normally use camelCase for our chpl code

Reply:
Fixed